.gitignore

```diff
@@ -2,4 +2,5 @@ vendor/
 storage/cache/aop/*
 storage/cache/api/*
 !storage/cache/aop/.gitkeep
-.env
\ No newline at end of file
+.env
+.idea/
```

Comment:
@strajeru I'm on the fence about this one. Personally I've added it to my global gitignore since it's local environment specific